jicofo-selector/src/main/kotlin/org/jitsi/jicofo/bridge/Bridge.kt

```diff
@@ -237,11 +241,15 @@ class Bridge @JvmOverloads internal constructor(
         return compare(this, other)
     }
 
-    /**
-     * Notifies this [Bridge] that it was used for a new endpoint.
-     */
+    /** Notifies this [Bridge] that it was used for a new endpoint. */
     fun endpointAdded() {
         newEndpointsRate.update(1)
+        endpoints.incrementAndGet()
+    }
+
+    fun endpointRemoved() = endpointsRemoved(1)
+    fun endpointsRemoved(count: Int) {
+        endpoints.addAndGet(-count);
```

Comment:
Suggest adding an assert that we haven't gone negative, to make sure we're not missing any adds or removes.